### PR TITLE
Controller & routes

### PR DESCRIPTION
**Pull request by amaachiaaaaa**
few updates on the controllers and created the route

### CODE REVIEW

File: controllers/achievementcontroller.js
Comment (Racheal777) [inline controllers/achievementcontroller.js:1]: Check this well   `const User = await user.findById(value, user);`
In the get one, update and delete, you have to first get all achievements that belong to the user first and then you can find by the particular achievementId

This applies to all models

File: routes/skillsroute.js
Comment (Racheal777) [inline routes/skillsroute.js:1]: check this code well, you are missing something
skillRouter.patch('/education/:id:educationStatus',updateSkill)

why is skills route path education

File: routes/userProfileroute.js
Comment (Racheal777) [inline routes/userProfileroute.js:1]: why is the user profile route path education

**Review by Racheal777 — COMMENTED** [review]
check on the comment and fix them

File: controllers/achievementcontroller.js
Comment (cameydanchi) [reply controllers/achievementcontroller.js:1]: Rachel i dont get what you are trying to say

**Pull request merged**